src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/hmi/rc_on_remote_control_settings_notification.cc

```diff
@@ -87,35 +87,34 @@ void RCOnRemoteControlSettingsNotification::DisallowRCFunctionality() {
 void RCOnRemoteControlSettingsNotification::Run() {
   SDL_LOG_AUTO_TRACE();
 
-  if (!(*message_)[app_mngr::strings::msg_params].keyExists(
-          message_params::kAllowed)) {
+  if ((*message_)[app_mngr::strings::msg_params].empty()) {
     SDL_LOG_DEBUG("Notification is ignored due to \"allow\" parameter absense");
```

Comment:
```suggestion
    SDL_LOG_DEBUG("Notification is ignored due to absence of any parameters");
```
Since both parameters are non-mandatory

Reply:
@ShobhitAd fixed in https://github.com/smartdevicelink/sdl_core/pull/3749/commits/57c6318e145a7ec01f9adc9d28b5549d45d1fa1a